pkg/status/status.go

```diff
@@ -439,25 +440,27 @@ func (NoOpReporter) StartReporting() <-chan struct{} {
 // of any non default Opsrc in the cluster to indicate that the cluster is not upgradable
 // to future versions.
 func CheckOperatorUpgradeablity(kubeClient client.Client) (bool, error) {
-	clusterHasOpsrc, err := isNonDefaultOpsrcPresent(kubeClient)
+	count, err := nonDefaultMarketplaceCRCount(kubeClient)
 	if err != nil {
 		return false, err
 	}
-	if clusterHasOpsrc {
+	metrics.RegisterCustomResource(metrics.ResourceTypeOpsrc, float64(count))
+	if count > 0 {
 		return false, nil
 	}
 	return true, nil
 }
 
-func isNonDefaultOpsrcPresent(kubeClient client.Client) (bool, error) {
+func nonDefaultMarketplaceCRCount(kubeClient client.Client) (int, error) {
 	opsrcs := &v1.OperatorSourceList{}
+	nonDefaultOpsrcCount := 0
 	if err := kubeClient.List(context.TODO(), &client.ListOptions{}, opsrcs); err != nil {
-		return false, err
+		return nonDefaultOpsrcCount, err
 	}
 	for _, opsrc := range opsrcs.Items {
 		if !defaults.IsDefaultSource(opsrc.Name) {
-			return true, nil
+			nonDefaultOpsrcCount++
```

Comment:
Why do we need to keep a count at all? Why can't we just have knowledge of if they exist on the cluster?

Reply:
I do not have a strong preference, but we lose information regarding how many Marketplace CRs by going binary. If you can explain potential benefits, I'll be happy to make the change.

Reply:
I actually think it's beneficial to have a count coz it enhances the UX for an admin. 

If it is binary: 
* Sees the alert
* Deletes an opsrc 
* Sees the alert still exists (but I already deleted a custom resource)

If it is a count: 
* Sees the alert
* Deletes a CR and sees the count go down 
* Alert tells the admin there's more CRs to be deleted.